### DISCUSSION TITLE
Client Token is empty on Facebook Dashboard / Advanced / Security / Client Token

### DISCUSSION THREAD

**Discussion by julianD77** [thebergamo/react-native-fbsdk-next#245] [Q&A]
Hi

Can anyone please help us to understand why our App does not seem to have a `Client Token` set (required for v8+ of this SDK, Facebook SDKs 13+) when viewing our App on the Facebook for Developers Dashboard?

From the Facebook docs we should see the token here: `Settings > Advanced > Security > Client token`

But our main Production App does not have any value in that field. Our Development App does however have a token set in that field. 

AFAIK this is a token that the Facebook for Developers console should just present and it is not something that we need to generate ourselves.

Does anybody have any ideas on this?

**Comment by mikehardy** [comment]
That is very interesting - I have not heard of that case before, all my apps had a token in the facebook developer dashboard web site, and as you mention: yes they are required now. This seems like something for facebook support directly and/or stackoverflow though, as we haven't seen it here yet.

Until you have an access token I don't believe you'll be able to use the new version here unfortunately. I would be interested in hearing whatever Facebook support tells you though, and it might help future people
